### PR TITLE
Changed shex example rdf and images

### PR DESCRIPTION
**Pull request by rajaram5**
I did minor changes to shex  and example RDF with `rdf:type` and replaced `prov` property with `sio`

### CODE REVIEW

**Comment by pabloalarconm** [comment]
All changes seems to be fine, im agree to merge it.

File: shex/sio-model/patientStatusDeadDateShape.shex [hunk 1]
```diff
@@ -5,7 +5,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 
 :personShape IRI {
-  a [obo:NCBITaxon_9606];
+  a [sio:SIO_000498];
   sio:SIO_000228 @:personRoleShape;
    sio:SIO_000217 @:deadDateQualityShape
```
Comment (sxzhang1201) [inline shex/sio-model/patientStatusDeadDateShape.shex:10]: Please remove extra space.

File: example-data/turtle/sio-model/diseaseHistoryAndDiagnosis.ttl [hunk 2]
```diff
@@ -18,18 +18,18 @@
...
 :process_1 a sio:SIO_001001 ;
   sio:SIO_000229 :output_1;
-  prov:startedAtTime  "2017-10-18T13:00:00"^^xsd:dateTime .
+  sio:SIO_000680  "2017-10-18T13:00:00"^^xsd:dateTime .
   
 :process_2 a sio:SIO_000006 ;
   sio:SIO_000229 :output_2;
-  prov:wasInformedBy :process_1;
-  prov:startedAtTime  "2015-10-18T13:00:00"^^xsd:dateTime .
+  sio:SIO_000230 :output_1; # has input
```
Comment (sxzhang1201) [inline example-data/turtle/sio-model/diseaseHistoryAndDiagnosis.ttl:25]: Are you sure that you want to add a comment here? (Also in ShEx)

**Review by sxzhang1201 — CHANGES_REQUESTED** [review]
The ShEx are syntactical correct except for minor changes:

- remove extra space;
- keep or remove comments?
- Remove **Prefix** of `prov` in all .ttl and .shex

File: example-data/turtle/sio-model/diseaseHistoryAndDiagnosis.ttl (hunk 2 above)
Comment (rajaram5) [reply example-data/turtle/sio-model/diseaseHistoryAndDiagnosis.ttl]: Not sure if we can keep them or remove them. But I removed it for now

File: example-data/turtle/sio-model/carePathway.ttl [hunk 3]
```diff
@@ -12,4 +10,4 @@
   sio:SIO_000356 :process_ .
 
 :process_ a obo:NCIT_C142427, obo:NCIT_C159705, sio:SIO_000006 ;
-    prov:startedAtTime "2020-10-18T13:00:00"^^xsd:dateTime .
```
Comment (sxzhang1201) [inline example-data/turtle/sio-model/carePathway.ttl:15]: You have different indentations within one document. We use four-space indentation in old versions.

**Review by sxzhang1201 — APPROVED** [review]
Looks good but the indentation needs to be consistent within a file.

File: example-data/turtle/sio-model/carePathway.ttl (hunk 3 above)
Comment (sxzhang1201) [reply example-data/turtle/sio-model/carePathway.ttl:15]: It is okay that two-space indentation will be used :D . But within one file, the indentations should be consistent.

Comment (rajaram5) [reply example-data/turtle/sio-model/carePathway.ttl:15]: I fixed it. Lets see if we can add a checkers for it in gihub

**Pull request merged**